packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -647,6 +647,12 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     this._stackedData.forEach((singleStackedData: Array<any>, index: number) => {
       graph.push(
         <React.Fragment key={`${index}-graph-${this._uniqueIdForGraph}`}>
+          <defs>
```

Comment:
can we make adding this def block also conditional based on the enableGradient prop.
This is causing our existing snapshot tests to break.

Reply:
Updated this def block to be conditional based on the enableGradient prop.